lib/cesiumGeometryToGltfPrimitive.js

```diff
@@ -1,10 +1,89 @@
 'use strict';
+var findAccessorMinMax = require('./findAccessorMinMax');
 var writeAccessor = require('./writeAccessor');
+var mergeBuffers = require('./mergeBuffers');
+var uninterleaveAndPackBuffers = require('./uninterleaveAndPackBuffers');
+var numberOfComponentsForType = require('./numberOfComponentsForType');
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+var WebGLConstants = Cesium.WebGLConstants;
+
 
 module.exports = cesiumGeometryToGltfPrimitive;
 
+function createAttributeSemantic(gltf, primitive, semantic, attributeLength) {
+    var type;
+    var bytesPerComponent;
+
+    switch (semantic.replace(/([0-9]|_)/g, '')) {
+        case 'POSITION':
+            type = 'VEC3';
+            bytesPerComponent = 8;
+            break;
```

Comment:
While the Cesium geometry positions are doubles, they should be converted back to floats for gltf `bytesPerComponent = 4`.